src/bjointsp/heuristic/control.py

```diff
@@ -112,7 +116,7 @@ def objective_value(overlays, print_info=False):
 
     # minimize total delay
     elif obj == objective.DELAY:
-        value = total_delay
+        value = total_delay+vnf_delays
```

Comment:
One more, important thing, I just noticed. There are 5 possible objectives for B-JointSP: Combined (default), over-subscription only, number of changed instances only, resources only, delay only.

You should add the `vnf_delay` to the total delay earlier, right after the place where it's calculated (currently line 79 or so). Currently, the vnf_delay is only added if the delay minimization objective is selected, but for example not for the default combined objective.